### DISCUSSION TITLE
Welcome to SnowAlert Discussions!

### DISCUSSION THREAD

**Discussion by sfc-gh-afedorov** [snowflakedb/SnowAlert#608] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smitthakkar1** [comment]
👋 @sfc-gh-afedorov ,

Thanks for putting this together. I have few general question. I am trying to use this library but it seems a little old. Last update was more than an year ago. I'm curious to learn about future plans if you have any

Also, do I need to contact snowflake support in order to get SnowAlert working ?

**Reply by sfc-gh-afedorov** [reply]
We've been updating the [v1.10.0 branch](https://github.com/snowflakedb/SnowAlert/tree/v1.10.0) which builds [RC releases](https://hub.docker.com/r/snowsec/snowalert/tags?page=1&ordering=last_updated) as we figure out how a Snowflake-hosted version of SnowAlert will integrate with features yet to be announced. We'll release one last ECS-based v1.10 here and keep it patched when we are ready with a plan to transition to the native version.

The future Snowflake Tasks based version will be in a separate repository in Snowflake-Labs and will be compatible with what's written in [SnowAlert documentation](https://docs.snowalert.com/) and interfaces like [data.alerts](https://github.com/snowflakedb/SnowAlert/blob/master/src/scripts/installer-queries/data-views.sql.fmt#L27).

**Reply by smitthakkar1** [reply]
Thanks for the update Andrey !
